README.md

```diff
@@ -2,6 +2,7 @@ fc
 ==
 
 [![](https://travis-ci.org/bitshares/bitshares-fc.svg?branch=master)](https://travis-ci.org/bitshares/bitshares-fc) 
+[https://github.com/bitshares/bitshares-fc/workflows/Github%20Autobuild/badge.svg?branch=master](https://github.com/bitshares/bitshares-fc/actions?query=branch%3Amaster)
```

Comment:
What about something more like this instead of the full link the 2 times?

[![](https://travis-ci.org/bitshares/bitshares-fc.svg?branch=master)](https://travis-ci.org/bitshares/bitshares-fc) 
[Github Build](https://github.com/bitshares/bitshares-fc/actions?query=branch%3Amaster)

Reply:
Fixed. The first URL is actually for a graphical badge, but I messed up the format.
Note that it's a broken image until we have a build in master.